groovy/groovy.editor/src/org/netbeans/modules/groovy/editor/completion/inference/TypeInferenceVisitor.java

```diff
@@ -99,6 +99,8 @@ public void visitField(FieldNode node) {
         if (sameVariableName(leaf, node)) {
             if (node.hasInitialExpression()){
                 guessedType = deriveExpressonType(node.getInitialExpression());
+            } else if (node.getType() != null) {
+                    guessedType = node.getType();
```

Comment:
Q: would `GroovyUtils.findInferredType()` in addition to getType() bring some benefits here ?

Reply:
I don't think so. In the time the metadata are null in this case.